plugins/modules/cloud/scaleway/scaleway_container.py

```diff
@@ -0,0 +1,417 @@
+                                                                      creation_response.json)
+            api.module.fail_json(msg=msg)
+
+        wait_to_complete_state_transition(api=api, resource=creation_response.json, stable_states=STABLE_STATES)
+        response = api.get(path=api.api_path + "/%s" % creation_response.json["id"])
+        return changed, response.json
+
+    target_cn = cn_lookup[wished_cn["name"]]
+    patch_payload = resource_attributes_should_be_changed(target_cn=target_cn,
+                                                          wished_cn=playload_cn,
+                                                          verifiable_mutable_attributes=VERIFIABLE_MUTABLE_ATTRIBUTES,
+                                                          mutable_attributes=MUTABLE_ATTRIBUTES)
+
+    if not patch_payload:
+        return changed, target_cn
+
+    changed = True
+    if api.module.check_mode:
+        return changed, {"status": "Container attributes would be changed."}
+
+    cn_patch_response = api.patch(path=api.api_path + "/%s" % target_cn["id"],
+                                  data=patch_payload)
+
+    if not cn_patch_response.ok:
+        api.module.fail_json(msg='Error during container attributes update: [{0}: {1}]'.format(
+            cn_patch_response.status_code, cn_patch_response.json['message']))
+
+    wait_to_complete_state_transition(api=api, resource=target_cn, stable_states=STABLE_STATES)
+    response = api.get(path=api.api_path + "/%s" % target_cn["id"])
+    return changed, response.json
+
+
+state_strategy = {
+    "present": present_strategy,
+    "absent": absent_strategy
+}
+
+
+def core(module):
+    region = module.params["region"]
+    wished_container = {
+        "state": module.params["state"],
+        "namespace_id": module.params["namespace_id"],
+        "name": module.params["name"],
+        "description": module.params['description'],
+        "min_scale": module.params["min_scale"],
+        "max_scale": module.params["max_scale"],
+        "environment_variables": module.params['environment_variables'],
+        "secret_environment_variables": module.params['secret_environment_variables'],
+        "memory_limit": module.params["memory_limit"],
+        "timeout": module.params["container_timeout"],
+        "privacy": module.params["privacy"],
+        "registry_image": module.params["registry_image"],
+        "max_concurrency": module.params["max_concurrency"],
+        "protocol": module.params["protocol"],
+        "port": module.params["port"],
+        "redeploy": module.params["redeploy"]
+    }
+
+    api = Scaleway(module=module)
+    api.api_path = "containers/v1beta1/regions/%s/containers" % region
+
+    changed, summary = state_strategy[wished_container["state"]](api=api, wished_cn=wished_container)
+
+    module.exit_json(changed=changed, container=filter_sensitive_attributes(summary, SENSITIVE_ATTRIBUTES))
```

Comment:
Here the return variable is not called `data`, but `container`.

Reply:
I have made the correction on all modules to remove the useless `data` level in the returned structure